pkg/telemetry/metric_db.go

```diff
@@ -0,0 +1,13 @@
+// Copyright 2023 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package telemetry
+
+type MetricsDB interface {
+	// CreateSchema creates table schemas to the provided database.
```

Comment:
We may need to consider more use cases:
Delete Schema
Update Schema
Delete Data
Get/Retrieve Data

Reply:
CLI is producer here and not consuming the data from DB. Telemetry plugin would be consuming the data from DB to send it to data lake. So for now `CreateSchema` should be sufficent, we would other operations if needed in future.